### DISCUSSION TITLE
Problem with automatic data import into PIMCore via DataHub: Need help setting up Messenger queue in Symfony

### DISCUSSION THREAD

**Discussion by jeanmermozeffi** [pimcore/pimcore#16831] [Data-Hub]
Hello Team, My client needs to import a huge amount of data into PIMCore. I added the import bundle, but I cannot do the automatic import under DataHub. Ask to help me in detail. This is where I need to change the Symphony configuration settings.
pimcore_data_importer:
messenger_queue_processing:
enabled: true
Request to help me resolve the import functionality

**Comment by Maksonis** [comment]
Two more things to add:
1. Messages are dispatched via pimcore_data_import transport. So make sure, you have workers processing this transport when activating the messenger based queue processing.
2. Cron execution:
`* * * * * php /home/project/www/bin/console datahub:data-importer:execute-cron`

**Reply by jeanmermozeffi** [reply]
Good morning
Thank you for your reply.
For automatic processing, I was forced to execute on each command.
However, I just solved the problem by adding "pimcore_data_import" in the supervisors command

[program:messenger-consume]
command=php /var/project/www/bin/console messenger:consume "pimcore_data_import" pimcore_core pimcore_maintenance pimcore_scheduled_tasks pimcore_image_optimize pimcore_asset_update pimcore_search_backend_message --memory-limit=500M --time-limit=3600
environment=MESSENGER_CONSUMER_NAME= consumer-%(program_name)s-%(process_num)02d
numprocs=1
startupsecs=0
autostart=true
autorestart=true
process_name=%(program_name)s_%(process_num)02d
stdout_logfile=/dev/fd/1
stdout_logfile_maxbytes=0
redirect_stderr=true

the "pimcore_data_import" was missing in the config file.
